cmd/security-secrets-setup/main.go

```diff
@@ -69,24 +75,52 @@ func main() {
 		return
 	}
 
-	if configFile == "" {
-		// run with other options for pki-init
-		statusCode, err := dispatcherInstance.run()
-		if err != nil {
-			setup.LoggingClient.Error(err.Error())
-		}
+	if checkIfMultipleSubcommands() {
```

Comment:
we don't need this anymore because this case is handled by two different cases below:

- the first subcommand is a known subcommand in which we know it doesn't accept args and fails in the `switch`
- the first subcommand is an unknown subcommand which is handle in the `if !found` ...

Reply:
ok